pages/contact.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react';
+import NavbarBootstrap from '../components/NavbarBootstrap';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Container from 'react-bootstrap/Container';
+
+class Contact extends Component {
+    render() {
+        return (<div>
+            <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous"/>
+            <NavbarBootstrap isAuthenticated={false}/>
```

Comment:
hmm we might not want to pass in a prop of `isAuthenticated={false}` into the navbar cause you could go to contact from an authenticated state... we need a way of determining that, cant just send hard coded false value here i dont think

Reply:
Discussed with @sdevalapurkar . We found another instance of this code in `index.js`. We found this as a potential solution: https://www.npmjs.com/package/cookie-auth . And this may also fix the issue where going back after logout logs the user back in without logging in

Reply:
Not in the scope of this PR. See: https://github.com/sdevalapurkar/smart-brake-monitor/projects/1#card-22710353